Sources/PT.PM.Cli.Common/CliParametersNormalizer.cs

```diff
@@ -44,7 +44,7 @@ public bool Normalize(string[] args, out string[] outArgs)
                 if (!args[argInd].StartsWith("-"))
                 {
                     error = true;
-                    Logger.LogError(new ArgumentException($"Incorrect argument", args[argInd]));
+                    NormalizationErrors.Add(new ArgumentException($"Incorrect argument", args[argInd]));
```

Comment:
What is the meaning of  `NormalizationErrors`? Why are they named in such a way?

Reply:
I got it. Since they are included in `CliParametersNormalizer`, I suggest to rename this property to `Errors`.